apps/web/nuxt.config.ts

```diff
@@ -174,7 +174,7 @@ export default defineNuxtConfig({
   },
   runtimeConfig: {
     public: {
-      apiUrl: process.env.API_URL ?? 'http://localhost:8181',
+      apiUrl: (process.env.API_URL ?? 'http://localhost:8181').replace(/[/\\]+$/, ''),
```

Comment:
we could write a helper here that gets us the api url and that can be tested.

Reply:
test what happens if we have a / at the end
test what happens if we have two slashes at the end
test what happens if we dont have a / at the end

Reply:
@maxiroellplenty just pushed the changes